index.html

```diff
@@ -4,43 +4,193 @@
+                        <img src="../img/Counter.png">
+                        &nbsp;
+                        <span>2015</span>
+                    </li> 
+                </p>
+            </div>
+            <div class ="flex-item flex-item-4">
+                <p class="project-content-4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class ="flex-item flex-item-5">
+                <img class="project-content-5" src="../img/html-css-js.png">
+            </div>
+            <div class ="flex-item flex-item-6"><button>See Project</button></div>
+        </div>
+
+
+
+    <div class="grid-item flex-container">
+        <div class ="flex-item flex-item-1"><h1 class="about-title">About <br> Myself</h1></div>
+        <div class ="flex-item flex-item-2">
+            <p class ="about-paragraph">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.   
+            </p> 
+        </div>
+        <div class ="flex-item flex-item-3">
+            <div class="about-connect">
                 LET’S CONNECT
             </div>
-            <br>
-            
-            <ul class="network-icons">
+        </div>
+
+        <div class="flex-item flex-item-4" >
+            <ul class="about-network-icons">
                 <li><a href="https://twitter.com" target="_blank"><img src="img/TwitterIcon.png" alt=""></a></li>
                 <li><a href="https://linkedin.com" target="_blank"><img src="img/LinkedinIcon.jpg" alt=""></a></li>
                 <li><a href="https://medium.com" target="_blank"><img src="img/MediumIcon.jpg" alt=""></a></li>
                 <li><a href="https://github.com" target="_blank"><img src="img/GithubIcon.jpg" alt=""></a></li>
                 <li><a href="https://youtube.com" target="_blank"><img src="img/HandIcon.jpg" alt=""></a></li>
             </ul>
-        </section>
+        </div>
+        <div class ="flex-item flex-item-5"><button class="flex-item flex-item-5">Get My Resume</button></div>
+        <div class ="flex-item flex-item-6">
+            <span class="about-language-title">
+                <span><h1 class ="about-languages">Languages</h1></span>
+                <span class="bottom"></span>
+            </span>
+        </div>
+        <div class ="flex-item flex-item-7">
+            <img src="/img/JS-List.png">
+            <img src="/img/HTML-List.png">
+            <img src="/img/CSS-List.png">
+            <br>
+            <br>
+        </div>
 
+        <div class ="flex-item flex-item-8"> 
+            <span class="about-language-title-1">
+                <span><h1 class ="about-languages">Frameworks</h1></span>
+                <span class="right"></span>
+            </span>
+            <br>
+            <hr>
```

Comment:
- [ ] [REQUIRED] The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li` and the technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a `li`. This essential for accessibility scores especially for those that use `keyboard tab key` to navigate the page.